app/assets/stylesheets/custom.css.scss

```diff
@@ -74,3 +74,13 @@ body {
 .create-item-form {
   width: 80%;
 }
+
+.item-create {
+  background: rgba(43,58,223,0.2);
+  width: 500px;
+  border: 1em;
+  border-color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.